tests/languages/test_hindi.py

```diff
@@ -62,6 +64,18 @@
     "मुट्ठ",
     "पागल",
     "हरामखोर",
+    "बहनचोद",
+    "गांडू",
+    "सूअर",
+    "हरामी",
+    "हराम जादा",
+    "लंड",
+    "कुतिया",
+    "कमीना",
+    "कमिनी",
```

Comment:
This spelling looks wrong to me. It should be " कमीनी "

Reply:
@Sharingan-Coder01 okay, I'll make the required changes